ebpf.mk

```diff
@@ -1,6 +1,6 @@
 SHELL=/bin/bash -o pipefail
-DEST_DIR=/dist
-LINUX_HEADERS=$(shell dnf list kernel-devel | awk '/^kernel-devel\..*/{print "/usr/src/kernels/"$$2".x86_64"}')
+DEST_DIR?=/dist
+LINUX_HEADERS=$(shell rpm -q kernel-devel --last | awk -F'kernel-devel-' '{print "/usr/src/kernels/"$$2}')
```

Comment:
Any reason for this change?

Also, this doesn't seem to work well for me:

```
[root@3fc70c7d8fa8 /]# LINUX_HEADERS=$(rpm -q kernel-devel --last | awk -F'kernel-devel-' '{print "/usr/src/kernels/"$2}')
[root@3fc70c7d8fa8 /]# echo $LINUX_HEADERS 
/usr/src/kernels/4.14.14-200.fc26.x86_64 Fri Jan 26 10:33:03 2018
```

so in the includes you get a lot of weird paths:

```
	 -I /usr/src/kernels/4.14.14-200.fc26.x86_64/arch/x86/include -I /usr/src/kernels/4.14.14-200.fc26.x86_64/arch/x86/include/generated -I /usr/src/kernels/4.14.14-200.fc26.x86_64/include -I /usr/src/kernels/4.14.14-200.fc26.x86_64/include/generated/uapi -I /usr/src/kernels/4.14.14-200.fc26.x86_64/arch/x86/include/uapi -I /usr/src/kernels/4.14.14-200.fc26.x86_64/include/uapi  -I Fri/arch/x86/include -I Fri/arch/x86/include/generated -I Fri/include -I Fri/include/generated/uapi -I Fri/arch/x86/include/uapi -I Fri/include/uapi  -I Jan/arch/x86/include -I Jan/arch/x86/include/generated -I Jan/include -I Jan/include/generated/uapi -I Jan/arch/x86/include/uapi -I Jan/include/uapi  -I 26/arch/x86/include -I 26/arch/x86/include/generated -I 26/include -I 26/include/generated/uapi -I 26/arch/x86/include/uapi -I 26/include/uapi  -I 10:24:02/arch/x86/include -I 10:24:02/arch/x86/include/generated -I 10:24:02/include -I 10:24:02/include/generated/uapi -I 10:24:02/arch/x86/include/uapi -I 10:24:02/include/uapi  -I 2018/arch/x86/include -I 2018/arch/x86/include/generated -I 2018/include -I 2018/include/generated/uapi -I 2018/arch/x86/include/uapi -I 2018/include/uapi \
```

Reply:
Regarding the `DEST_DIR?=/dist`, it's because I'm using `ebpf.mk` outside a Docker container.

For the LINUX_HEADERS definition, `dnf list` can return uninstalled packages (though this can be modified with --installed), is somewhat longer.

Reply:
With the new changes it's the `rpm` version is now longer but ok, showing only installed packages is an advantage.